data_processing/tests/test_eia_functions.py

```diff
@@ -0,0 +1,539 @@
+import pytest
+from eia import fuel_type_code
+from eia import mover_database
+from eia import generation
+from eia import university_top_producers
+from eia import energy_type_breakdown
+from eia import university_top_renewables
+from eia import usage
+from eia import capacity_factors
+from eia import university_capacity
+from eia import sources
+from eia import split_dictionary
+from eia import plot_data
+from eia import plot_energy_change
+path = 'eia_sample_'
```

Comment:
```suggestion
path = "./"
fileprefix= "eia_sample_"
```
The suggestions made in ``eia.py`` should be reflected here.